src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -451,12 +371,7 @@ private function setRealFormat($format)
         $this->formatLineCount = substr_count($this->format, "\n");
     }
 
-    /**
-     * Sets the progress bar maximal steps.
-     *
-     * @param int $max The progress bar max steps
-     */
-    private function setMaxSteps($max)
+    private function setMaxSteps(int $max)
     {
         $this->max = max(0, (int) $max);
         $this->stepWidth = $this->max ? Helper::strlen($this->max) : 4;
```

Comment:
beware, `Helper::strlen` expect a string. It's not an issue, because currently the method is not typehinted, but we dont respect the contract.